tasks/init/package.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file
+ * Handle operations pertaining to the package.json manifest file.
```

Comment:
Is this for building a file from the init command? I'm a bit confused.

Reply:
No, the `init` command assumes there already is a `package.json` file present and updates it to add all of those commands that have to live in each project (the ones you called out during the demo a few weeks back). I did start out with some logic here where if the file didn't exist `init` could create it for you, but I ran into some roadblocks and my wheels were spinning for a minute, so I scaled it back. I do want to pursue this in the future, however, as I think two steps are better than three.

In their current state in this PR, the setup instructions include a step to install Calliope in your project first. That will generate `package.json` for you if it doesn't already exist, so it's a safe assumption to make. Regardless of that, if `package.json` is not found when you run `init`, that error state is handled and reported in the output to the user with potential fixes.

Does that answer your question?